README.md

```diff
@@ -4,20 +4,20 @@ Make sure to upload your files containing user data to a secure folder!
 
 ### Setup a protected storage folder
 
-1. Create a file protected "File Storage" record
-2. Create a folder /protected/userimport in your webroot
-3. Create a "Filemount" record for the folder
-4. Add the Filemount to a "Backend Usergroup"
+1. Create a folder /protected/user_upload/userimport in your webroot.
+2. Create a protected "File Storage" record and set "protected/" as base path and disallow public access.
+3. Create a "Filemount" record for the folder protected/user_upload/userimport.
+4. Add the Filemount to a "Backend Usergroup" to allow access for non admin users.
 
 ![File storage record](filestorages.png)
 
 ### Add your storage folder to the module configuration
 
-Module configuration in you extension manager.
+Module configuration in this Extension Manager.
```

Comment:
Why "this" Extension Manager? There is only one, so I would write "the Extension Manager"